htdocs/js/pages/entry/new.js

```diff
@@ -673,6 +673,7 @@ var postForm = (function($) {
         initSticky(entryForm);
 
         $("#js-usejournal").triggerHandler("change");
+        $("#js-entrytime-autoupdate").triggerHandler("click");
```

Comment:
Oh nice, I am guessing this is to make sure that when the checkbox is checked by default, the associated actions / states come into play right?